<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="btnTrainModel">
     <property name="text">
      <string>Train model</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="pteWeightsInform"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnResetBins">
     <property name="text">
      <string>Reset Bins</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Weights Information:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="MolusceTableWidget" name="tblReclass">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Factor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Range min</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Range max</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Number of intervals</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bin breaks</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MolusceTableWidget</class>
   <extends>QTableWidget</extends>
   <header>molusce.moluscetablewidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
